<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="Peach">Project Peach</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>311</width>
      <height>187</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="pcd_label">
       <property name="text">
        <string>Select Point Cloud Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pcd_button">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="treecylinder_label">
       <property name="text">
        <string>Select Tree Cylinders</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cylinder_button">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="treedata_label">
       <property name="text">
        <string>Select Tree Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="treedata_button">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>571</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="prunepercentage_label">
       <property name="text">
        <string>Prune Percentage</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="prune_slider">
       <property name="minimum">
        <number>20</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
       <property name="sliderPosition">
        <number>40</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>20%                                                                 Default                                                                60%</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>251</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Select a pruning method:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="prunemethods_combobox">
       <property name="currentText">
        <string/>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Open Center</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Distance</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Height</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="prune_button">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Prune a Peach Tree!</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>633</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPeachtree_Pruner">
    <property name="title">
     <string>Peachtree Pruner</string>
    </property>
   </widget>
   <addaction name="menuPeachtree_Pruner"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
